.github/workflows/main_locatecontractorapi.yml

```diff
@@ -9,7 +9,7 @@ on:
     branches:
       - main
   pull_request:
-    types: [opened, synchronize, reopened, closed]
+    types: [closed]

```

Comment:
These changes were already mad in another PR, where are the doubled up? This could cause conflicts later